doc/user/content/ops/troubleshooting.md

```diff
@@ -10,58 +10,120 @@ aliases:
 ---
 
 You can use the queries below for spot debugging, but it may also be
-helpful to make them more permanent tools by creating them as views for `SUBSCRIBE`ing or materialized views to read more efficiently. Note that the
-existence of these additional views may itself affect performance.
+helpful to monitor their output by `SUBSCRIBE`ing to their change stream, e.g.,
+by issuing a command `COPY (SUBSCRIBE (<query>) TO stdout`.
```

Comment:
```suggestion
by issuing a command `COPY (SUBSCRIBE (<query>)) TO stdout`.
```

Should we link to the `SUBSCRIBE` docs instead or in addition?

Reply:
Done.